unit-tests/algo/d2rgb/test-reproduction.cpp

```diff
@@ -129,6 +129,34 @@ class custom_ac_logger : public ac_logger
     }
 };
 
+
+bool try_to_get_thermal_data( std::string dir,
+                              double hum_temp,
+                              const std::pair< double, double > & in_fx_fy,
+                              std::pair< double, double > & out_fx_fy )
+{
+    try
+    {
+        rs2_intrinsics raw_rgb_calib;
+        read_binary_file( dir.c_str(), "raw_rgb.calib", &raw_rgb_calib );
```

Comment:
`raw_rgb.calib` is the thermal table? if so, I think `thermal.table` is more understandable?

Reply:
no. its the krgb before thermal correction there is also "rgb_thermal_table" after that.

Reply:
so `rgb.calib` changed its meaning? It was Krgb but is now Kth? In that case, I think we should change its name.